src/Label2.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {get} from './constants'
+
+interface Props {
+  /** The color of the label */
+  variant?: LabelColorOptions
+  /** How large the label is rendered */
+  size?: LabelSizeKeys
+}
```

Comment:
We typically use `type` instead of `interface` for prop type definitions. Also, `LabelProps` (instead of `Props`) is clearer name IMO.
```suggestion
export type LabelProps = {
  /** The color of the label */
  variant?: LabelColorOptions
  /** How large the label is rendered */
  size?: LabelSizeKeys
}
```